io.catenax.shared.message_header/1.0.0/MessageHeaderAspect.ttl

```diff
@@ -0,0 +1,128 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.shared.message_header:1.0.0#> .
+@prefix ext-header: <urn:samm:io.catenax.message_header:1.0.0#> .
```

Comment:
Why is an external aspect referenced here? Will this be fixed before release?

Reply:
this was an oversight when exporting the file from the modelling editor. Another model was loaded in the editor and changing the namespace to a shared namespace did not propagate to all terms. Has since been fixed